tests/config/pytest_modules.yml

```diff
@@ -669,7 +681,7 @@ cooler/merge:
 
 cooler/zoomify:
   - modules/nf-core/cooler/zoomify/**
-  - tests/modules/nf-core/cooler/zoomify/**
+  - tests/software/cooler/zoomify/**
```

Comment:
This looks wierd to me

Reply:
I have no idea what happened there as well as in the other place... As mentioned, I'm still a bit new to git and nextflow. Can you guess what happened here?

Reply:
(So I can look out for it in the future?)

Reply:
I'm not really sure - you never should have to touch these files (they should be updated automagically by `nf-core tools`, although I guess in this case you had to manually edit them maybe when splitting the database update system?). Possibly a bad `merge` from master?

Reply:
Aha I see, in the future, I don't sync changes from my origin but instead, keep everything up to date with `nf-core tools`. Thx for being so helpful!